app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -22,11 +22,18 @@ sealed class RouteAction : Action {
     data class OpenWebsite(val url: String) : RouteAction()
     data class SystemSetting(val setting: SettingIntent) : RouteAction()
 
-    sealed class DialogAction(
-        val positiveButtonAction: Action? = null,
-        val negativeButtonAction: Action? = null
+    sealed class Dialog(
+        val positiveButtonActions: List<Action>? = null,
+        val negativeButtonActions: List<Action>? = null
     ) : RouteAction() {
-        class SecurityDisclaimerDialog(setUpAction: Action) : DialogAction(positiveButtonAction = setUpAction)
+        object SecurityDisclaimer : Dialog(positiveButtonActions = listOf(
+            RouteAction.SystemSetting(SettingIntent.Security)
+        ))
+        object UnlinkDisclaimer : Dialog(positiveButtonActions = listOf(
+            DataStoreAction.Reset,
+            SettingAction.Reset,
+            AccountAction.Reset
+        ))
```

Comment:
Hmm. I'm not sure I like that a `RouteAction` is the one place we unify these `Reset` actions.

I'd quite like to be able to send a single action that cascades these out, rather than have it hidden behind a dialog action.

Looks cool though.